sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -16,7 +32,7 @@ and
 [samples](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration/src/samples/java/com/azure/data/appconfiguration)
 
 ## Version 1.0.0-preview.5 (2019-10-11)
-For details on the Azure SDK for Java (September 2019 Preview) release refer to the [release announcement](https://aka.ms/azure-sdk-preview5-java).
+For details on the Azure SDK for Java (September 2019 Preview) release refer to the [release announcement](https://azure.github.io/azure-sdk/releases/2019-10-11/java.html).
```

Comment:
why are we changing the previously shipped aka.ms links?  Should these be reverted?

Reply:
After we released, then the link needed to be updated.  I think this is the strategy we are using now. @alzimmermsft